db/migrate/20210128123446_create_tasks.rb

```diff
@@ -0,0 +1,14 @@
+class CreateTasks < ActiveRecord::Migration[6.0]
+  def change
+    create_table :tasks do |t|
+      t.string :name
+      t.text :description
+      t.integer :author_id
+      t.integer :assignee_id
```

Comment:
```
belongs_to :author, class_name: 'User'
belongs_to :assignee, class_name: 'User', optional: true
```
this code represents an association between models (in DB also)
so you need to add migration with converting this fields into `Foreign Keys`
(yep, this info is missing in the lesson, sorry)